Userland/Libraries/LibGfx/Color.h

```diff
@@ -184,14 +184,11 @@ class Color {
 
     static Color from_lab(float L, float a, float b, float alpha = 1.0f);
     static Color from_xyz50(float x, float y, float z, float alpha = 1.0f);
+    static Color from_linear_srgb(float x, float y, float z, float alpha = 1.0f);
```

Comment:
The parameters to this are `red, green, blue`, not `x, y, z`.

Reply:
Oops, will change!